rom/dev/tests/rom_integration_tests/rv32_unit_tests.rs

```diff
@@ -6,15 +6,15 @@ use caliptra_hw_model::{BootParams, HwModel, InitParams};
 #[test]
 fn test_asm() {
     let rom = caliptra_builder::build_firmware_rom(&firmware::rom_tests::ASM_TESTS).unwrap();
-    let mut hw = caliptra_hw_model::new(BootParams {
-        init_params: InitParams {
+    let mut hw = caliptra_hw_model::new(
```

Comment:
This seems fine, but Hardware model could benefit of using Rust's builder pattern. HwModelBuilder builds HwModel https://rust-unofficial.github.io/patterns/patterns/creational/builder.html

Reply:
That's a good point. This could be a good place to use this in the future.